business_objects/tokenization.py

```diff
@@ -79,9 +81,14 @@ def get_doc_bin_progress(project_id: str) -> str:
 def get_doc_bin_table_to_json(
     project_id: str, missing_columns: str, record_ids: List[str] = None
 ) -> Any:
+    project_id = prevent_sql_injection(project_id, isinstance(project_id, str))
```

Comment:
If we follow an approach of sanitizing input possible given by users, we should here maybe also sanitize the missing_columns because the value can be set by users through attribute naming

Reply:
collected from database & prepared beforehand. Added comment as indicator